exercises/go-counting/README.md

```diff
@@ -22,8 +23,8 @@ marked "I" (ignored).  Empty spaces represent empty intersections.
 +----+
 |IOOI|
 |O  O|
-|O OI|
-|IOI |
+|O O |
```

Comment:
The README is generated using `configlet` based on [this description](https://github.com/exercism/problem-specifications/blob/master/exercises/go-counting/description.md)

From what I can see, this README did not need to change :slightly_smiling_face: